CONTRIBUTING.md

```diff
@@ -77,6 +77,44 @@ README for more details.
 - Once a pull request is good to go, the code reviewer is the one who merges
 it, then deletes the branch.
 
+## Front end architecture
+### CSS
+- The front end will use the Web Design Standard's components, imported through ruby, adding extra functionality that's missing.
+- Atomic CSS methodolgy will be used for layout, modifier classes.
+  - If a group of atomic classes is used more then once in a repeated pattern, separate it into it's own component
```

Comment:
i think this is a bit too rigid, would remove or rephrase to: "if you find yourself repeating HTML elements with the same set of atomic classes, consider refactoring UI into its own template (to be included across views)."